src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/AbstractVisualBasicCompletionProviderTests.vb

```diff
@@ -15,8 +15,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Completion.Complet
     Public MustInherit Class AbstractVisualBasicCompletionProviderTests
         Inherits AbstractCompletionProviderTests(Of VisualBasicTestWorkspaceFixture)
 
-        Public Sub New(workspaceFixture As VisualBasicTestWorkspaceFixture)
-            MyBase.New(workspaceFixture)
+        Protected Sub New()
         End Sub
```

Comment:
consider removing.

Reply:
I kept `protected` constructors within `abstract` types. Constructors for concrete types were removed in favor of the implicit `public` one.

Reply:
I think the compiler already emits the implicit ctor for abstract types as protected by default.

Reply:
TIL!

Reply:
➡️ Removed